src/services/admin/admin.tsx

```
@@ -0,0 +1,109 @@
+		if (!authToken) {
+			throw new Error('Authentication token not found');
+		}
+		const response = await axios.get(`${BASE_URL}/fields`, {
+			params: { context, contextType },
+			headers: {
+				accept: 'application/json',
+				Authorization: `Bearer ${authToken}`,
+			},
+		});
+		// Response is an array of field objects
+		return response.data.map((field: any) => ({
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)